### PR TITLE
Standardize key prefixes for translations

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -1100,8 +1100,8 @@
     "No": "No",
     "NoCreatedLibraries": "Seems like you haven't created any libraries yet. {0}Would you like to create one now?{1}",
     "NoNewDevicesFound": "No new devices found. To add a new tuner, close this dialog and enter the device information manually.",
-    "NoNextUpItemsMessage": "None found. Start watching your shows!",
-    "NoPluginConfigurationMessage": "This plugin has no settings to configure.",
+    "MessageNoNextUpItems": "None found. Start watching your shows!",
+    "MessageNoPluginConfiguration": "This plugin has no settings to configure.",
```
Comment (MrTimscampi) [inline src/strings/en-us.json:1104]: Why rename this one and not the surrounding ones?

Feels like all the placeholders should have a unified name.

Comment (dkanada) [reply src/strings/en-us.json:1104]: I was specifically only renaming keys here that incorrectly used the placeholders.

File: src/strings/en-us.json [hunk 2]
```diff
@@ -576,11 +576,11 @@
     "LabelAuthProvider": "Authentication Provider:",
     "LabelAutomaticallyRefreshInternetMetadataEvery": "Automatically refresh metadata from the internet:",
     "LabelBindToLocalNetworkAddress": "Bind to local network address:",
-    "LabelBindToLocalNetworkAddressHelp": "Optional. Override the local IP address to bind the http server to. If left empty, the server will bind to all availabile addresses. Changing this value requires restarting Jellyfin Server.",
+    "LabelBindToLocalNetworkAddressHelp": "Override the local IP address for the HTTP server. If left empty, the server will bind to all availabile addresses. Changing this value requires restarting Jellyfin Server.",
     "LabelBirthDate": "Birth date:",
     "LabelBirthYear": "Birth year:",
     "LabelBitrate": "Bitrate:",
-    "LabelBlastMessageInterval": "Alive message interval (seconds)",
+    "LabelBlastMessageInterval": "Alive message interval",
```
[the remark this answers is not in the document]
Comment (dkanada) [reply src/strings/en-us.json:583]: Maybe caps with a dash? I'm looking up usage now.

[the remark this answers is not in the document]
Comment (dkanada) [reply src/strings/en-us.json:583]: We can fix this one one Weblate.

**Pull request merged**